apps/publisher/modules/asset-api.js

```diff
@@ -314,6 +315,7 @@ var result;
      * @return The created asset or null if failed to create the asset
      */
     api.create = function (options, req, res, session) {
+        metrics.start("asset-api","create");
```

Comment:
Thanks @lahirus , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/2580f7b98d5e758340b17bc1d4be41ad8c1c218b)